content/spin/cache.md

```diff
@@ -7,31 +7,62 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/cache.md"
+> This document is provided as a reference for users wanting to diagnose problems or to reset Spin state. Don't modify the contents of these directories. Spin updates these directories as you issue the relevant commands on the command line.
 
-## Clearing the Registry Cache
+> No stability guarantees apply to the internal layout. It may change between Spin versions.
 
-Clearing the registry cache directory can be done by removing the `registry` directory entirely. The only direct effect of these actions is that Spin will have to pull again all component sources and static assets.
+## Base Directories
 
-### Viewing Registry Cache Files
+Spin uses similar layouts across Linux, MacOS and Windows platforms, but the paths to various areas of the user's home directory differ across the platforms. On this page, the following terms have the following meanings:
 
-> **Note:** The registry cache contains no user-serviceable parts; the only safe way to make changes to the contents of the cache is through the Spin command line. The only operation a user can safely undertake is to delete the `registry` directory.
+| Name          | Linux                                    | MacOS                                | Windows |
+|---------------------|------------------------------------------|--------------------------------------|-------------------|
+| `DATA_DIR`    | `$XDG_DATA_HOME` or `$HOME/.local/share` | `$HOME/Library/Application Support`, or `$HOMEBREW_PREFIX/etc/fermyon-spin` if installed using Homebrew  | `%LOCALAPPDATA%` or `%USERPROFILE%\AppData\Local` |
+| `CACHE_DIR`   | `$XDG_CACHE_HOME` or `$HOME/.cache`      | `$HOME/Library/Caches`               | `%LOCALAPPDATA%` or `%USERPROFILE%\AppData\Local` |
 
-All downloadable application files that are automatically cached can be found in the following file paths, depending on the operating system being used to run the Spin application:
+These directories are based on the [XDG specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), and specifically on the cross-platform implementation in the [Rust `dirs` crate](https://docs.rs/dirs/latest/dirs/).
 
-| Platform | Value | Example |
-| :--- | :--- | :--- |
-| Linux | `$XDG_CACHE_HOME` or `$HOME/.cache/spin/registry` | `/home/alice/.cache/spin/registry` |
-| macOS | `$HOME/Library/Caches/spin/registry` | `/Users/Alice/Library/Caches/spin/registry` |
-| Windows | `{FOLDERID_LocalAppData}/spin/registry` | `C:\Users\Alice\AppData\Local/spin/registry` |
+> If Spin cannot resolve a base directory as listed above, it falls back to `$HOME/.spin` (`%USERPROFILE%\.spin` on Windows).
 
-For example inspecting the local `spin/registry` cache on macOS, (using the `tree` command) would look like this:
+## Plugins
+
+Installed plugins are stored in `(DATA_DIR)/spin/plugins`.  A snapshot of the plugins registry is also stored under that directory at `(DATA_DIR)/spin/plugins/.spin-plugins`.
```

Comment:
It's an implementation detail that it's a git repo though, right?  From a user point of view it's just a bunch of plugins out there in the cloud?  Of course, this is the internals page... could we say something like "A snapshot of the plugins registry is also stored at ... This is structured as a Git repository."?

Reply:
The again, this page later talks about OCI 'registries' so maybe better to avoid that term... it sure doesn't help that we've overloaded every word in the dictionary... I'll do the repository change you suggest and we can always tweak later.  Thanks (and sorry for rambling)

Reply:
i like the changes you made -- its a registry but implemented in this repository 🥇